doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -1,17 +1,11 @@
-.. warning::
-    This page is under construction!
-
-.. _jobs-quickstart-under-construction:
+.. _jobs-quickstart:
 
 =================================
 Quickstart Using the Ray Jobs CLI
 =================================
 
 In this guide, we will walk through the Ray Jobs CLIs available for submitting and interacting with a Ray Job.
 
-.. note::
```

Comment:
Why the removal? Jobs is still in beta.

Reply:
Reverted this. Though, it is very odd we're recommending a beta.